CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ set(AMDGPU_TARGETS gfx900;gfx906;gfx908 CACHE STRING "List of specific machine t
 find_package(ROCM
              REQUIRED
              PATHS
-             /opt/rocm)
+             /opt/rocm $CMAKE_INSTALL_PREFIX)
```

Comment:
I'm not clear on what this is supposed to do.  Could you explain the reasoning behind this?

Reply:
Agreed this can be removed since CMAKE already put INSTALL_PREFIX as a search path.